shared/engine/index.platform.desktop.js

```diff
@@ -17,6 +19,26 @@ class NativeTransport extends TransportShared {
     super._connect_critical_section(cb)
     windowsHack()
   }
+
+  // Override Transport._raw_write -- see transport.iced in
+  // framed-msgpack-rpc.
+  _raw_write(msg, encoding) {
+    if (printRPCBytes) {
+      const b = Buffer.from(msg, encoding)
+      logger.debug('[RPC] Writing', b.length, 'bytes:', b.toString('hex'))
+    }
+    // $FlowIssue Flow doesn't see inherited methods.
```

Comment:
for posterity: this a feature we enable in `.flowconfig`, `munge_underscores=true`

Reply:
thanks, clarified comments!